### PR TITLE
[issue-8] Add spring-data-redis support

### LINKED ISSUE #8: Add instrument for spring-data-redis

### PR DESCRIPTION
**Pull request by ddcprg**
Fixes #8 

@malafeev please take a look

### CODE REVIEW

File: opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingRedisAsyncCommands.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing.contrib.redis.lettuce;
+package io.opentracing.contrib.redis.spring;
```
Comment (malafeev) [inline opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingRedisAsyncCommands.java]: why package changed? it is under lettuce.

File: opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingRedisCommands.java [hunk 2]
```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing.contrib.redis.lettuce;
+package io.opentracing.contrib.redis.spring;
```
Comment (malafeev) [inline opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingRedisCommands.java]: why package changed? it is under lettuce.

File: opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingStatefulRedisConnection.java [hunk 3]
```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing.contrib.redis.lettuce;
+package io.opentracing.contrib.redis.spring;
```
Comment (malafeev) [inline opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingStatefulRedisConnection.java]: why package changed? it is under lettuce.

File: opentracing-redis-lettuce/src/test/java/io/opentracing/contrib/redis/spring/TracingLettuceTest.java [hunk 4]
```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing.contrib.redis.lettuce;
+package io.opentracing.contrib.redis.spring;
```
Comment (malafeev) [inline opentracing-redis-lettuce/src/test/java/io/opentracing/contrib/redis/spring/TracingLettuceTest.java]: why package changed? it is under lettuce.

File: opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingRedisAsyncCommands.java (hunk 1 above)
Comment (ddcprg) [reply opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingRedisAsyncCommands.java]: mm, not sure... it's not supposed to have changed, let me revert this and the others

Comment (ddcprg) [reply opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingRedisAsyncCommands.java]: sorry for that, something funny happened in my IDE and I didn't notice it

**Comment by malafeev** [comment]
thanks @ddcprg

**Comment by malafeev** [comment]
`java-redis-client` version 0.0.7 with spring support is released.
It will be available in maven central during 24 hours.

**Pull request merged**